ckanext/switzerland/templates/package/read.html

```diff
@@ -1,5 +1,6 @@
 {% ckan_extends %}
 
+{% block primary_content_inner %}
```

Comment:
Overriding this whole block to remove the `tags` block, which would display keywords in all four languages.